sixtyfps_runtime/interpreter/dynamic_component.rs

```diff
@@ -439,9 +439,38 @@ impl<'id> ComponentDescription<'id> {
         if !core::ptr::eq((&self.ct) as *const _, component.get_vtable() as *const _) {
             return Err(());
         }
-        let x = self.custom_callbacks.get(name).ok_or(())?;
-        let sig = x.apply(unsafe { &*(component.as_ptr() as *const dynamic_type::Instance) });
-        sig.set_handler(handler);
+        if let Some(alias) = self
+            .original
+            .root_element
+            .borrow()
+            .property_declarations
+            .get(name)
+            .and_then(|d| d.is_alias.as_ref())
+        {
+            generativity::make_guard!(guard);
+            // Safety: we just verified that the component has the right vtable
+            let c = unsafe { InstanceRef::from_pin_ref(component, guard) };
+            generativity::make_guard!(guard);
```

Comment:
Is there one guard too many?

Reply:
the first guard is moved in the previous line and cannot be re-used.  I could have named them differently, but i thought it would not be necessary